src/package.json

```diff
@@ -19,13 +19,17 @@
   "dependencies": {
     "@internal/cache": "file:lib/cache",
     "@internal/log": "file:lib/log",
+    "@internal/middleware": "file:lib/middleware",
```

Comment:
I really think this should be something specific as `middleware` can mean anything. I.e. why isn't this `tracing-middleware`, unless you think the same component will be used to define other middleware components?